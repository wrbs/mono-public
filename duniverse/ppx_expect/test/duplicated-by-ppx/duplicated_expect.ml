[%%duplicate
  let%expect_test "apples and apples" =
    print_endline "buy apple";
    [%expect {| buy apple |}]
  ;;]
