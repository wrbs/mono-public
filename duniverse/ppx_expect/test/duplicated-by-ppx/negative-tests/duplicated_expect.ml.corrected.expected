[%%duplicate
  let%expect_test "apples and oranges" =
    print_endline "buy apple";
    [%expect {| buy apple |}]
  ;;]
